### PR TITLE
update current version of pion-1 and deps

### PR DESCRIPTION
**Pull request by sotnikov-s**

### CODE REVIEW

File: testnets/neutrontestnet/chain.json [hunk 1]
```diff
@@ -25,16 +25,16 @@
   },
   "codebase": {
     "git_repo": "https://github.com/neutron-org/neutron",
-    "recommended_version": "v4.0.0-rc3",
+    "recommended_version": "v4.2.2-testnet",
```
Comment (JeremyParish69) [inline testnets/neutrontestnet/chain.json:28]: Be sure to update the `versions` array

File: testnets/neutrontestnet/chain.json [hunk 2]
```diff
@@ -25,16 +25,16 @@
...
+      "v4.2.2-testnet"
     ],
     "cosmos_sdk_version": "0.50",
     "consensus": {
       "type": "cometbft",
-      "version": "0.38.7"
+      "version": "0.38.11"
     },
-    "cosmwasm_version": "0.52",
+    "cosmwasm_version": "0.51",
```
Comment (JeremyParish69) [inline testnets/neutrontestnet/chain.json:37]: Will need to update the `cosmwasm` property as well

File: testnets/neutrontestnet/chain.json (hunk 1 above)
Comment (sotnikov-s) [reply testnets/neutrontestnet/chain.json:28]: done

File: testnets/neutrontestnet/chain.json (hunk 2 above)
Comment (sotnikov-s) [reply testnets/neutrontestnet/chain.json:37]: done as well. please take another look 🙏  it must be all

**Pull request merged**